ERCS/erc-7699.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 7699
```

Comment:
```suggestion
eip: 7704
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
7704